runtime/autoload/remote/host.vim

```diff
@@ -177,19 +178,24 @@ endfunction
 
 " Registration of standard hosts
 
-" Python {{{
+" Python/Python3 {{{
 function! s:RequirePythonHost(name)
+  let ver = (a:name ==# 'python') ? 2 : 3
```

Comment:
I think this will break on Arch Linux, as `python` is an alias for `python3`. Maybe we should just check for `python2` and `python3`, but not `python`.